the test relies on `uname -m`, so it'll fail on non-i386 systems

diff --git a/testsuite/config-guess.data b/testsuite/config-guess.data
index 7f6f214..189b067 100644
--- a/testsuite/config-guess.data
+++ b/testsuite/config-guess.data
@@ -32,8 +32,6 @@ xtensa	2.6.15			Linux		ignored	xtensa-unknown-linux-gnu
 x86_64	1.0			MINGW		ignored	x86_64-pc-mingw32
 i386	1.0			MINGW		ignored	i386-pc-mingw32
 SX-8R	1.0			SUPER-UX	ignored	sx8r-nec-superux1.0
-i86xen	5.6			SunOS		ignored	i386-pc-solaris2.6
-i86pc	5.6			SunOS		ignored	i386-pc-solaris2.6
 IA64	6.0			Interix		ignored	ia64-unknown-interix6.0
 BePC	1.0			Haiku		ignored	i586-pc-haiku
 padre	2.6.15			Linux		ignored	sparc-unknown-linux-gnu
